test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,107 @@
+### Examples of Individual Steps
+As an alternative to the quick-start deployment bootstrapping instructions, you can also run the individual steps starting from a local directory containing the git repo as follows:
+
+1. Create the KubeStellar core infra on AWS. The infrastructure includes a VPC, security groups, EC2 instances, etc.
+
+    ```bash
+    cd test/scale-infra
+    ansible-playbook deploy_vpc_core.yaml -e "region=us-east-2 vpc_name=<vpc_name>"
+    ansible-playbook create-ec2.yaml -e "cluster_name=core region=us-east-2 vpc_name=<vpc_name> aws_key_name=mykey  num_masters=1 num_workers=2 instance_type=t2.xlarge arch=x86_64 ec2_image='<your-aws-ami>'"
```

Comment:
You might want to call out the variable "mykey" as well as the other two.

You might want to use shell variable syntax for your variables, so that readers do not have to edit long commands.

Reply:
Fixed. Added shell variable syntax for all the variables.